relay-server/src/actors/healthcheck.rs

```diff
@@ -2,45 +2,63 @@ use std::sync::atomic::{AtomicBool, Ordering};
 use std::sync::Arc;
 
 use actix::SystemService;
-use tokio::sync::{mpsc, oneshot};
 
 use relay_config::{Config, RelayMode};
 use relay_metrics::{AcceptsMetrics, Aggregator};
 use relay_statsd::metric;
-use relay_system::{compat, Controller};
+use relay_system::{
+    compat, Addr, AsyncResponse, Controller, FromMessage, Interface, Sender, Service,
+};
 
 use crate::actors::upstream::{IsAuthenticated, IsNetworkOutage, UpstreamRelay};
 use crate::service::REGISTRY;
 use crate::statsd::RelayGauges;
-use relay_system::{Addr, Service, ServiceMessage};
+
+#[derive(Clone, Copy, Debug)]
+pub enum IsHealthy {
+    /// Check if the Relay is alive at all.
+    Liveness,
+    /// Check if the Relay is in a state where the load balancer should route traffic to it (i.e.
+    /// it's both live/alive and not too busy).
+    Readiness,
+}
+
+/// Interface of the [`Healthcheck`] service.
+pub struct Healthcheck(IsHealthy, Sender<bool>);
```

Comment:
```suggestion
pub struct HealthCheck(IsHealthy, Sender<bool>);
```
just curious why not `HealthCheck`?

Reply:
No particular reason, should we change it everywhere? So far we've been using "healthcheck" as a single word in endpoints and code.

Reply:
yeah i saw it was before, btw https://english.stackexchange.com/questions/263518/should-i-use-health-check-or-healthcheck

Reply:
Added the change in https://github.com/getsentry/relay/commit/d88d4a6e3332ef6cde9f03c4e2bf495ab833f79a, which will go into a separate PR. Thanks for the suggestion!

Reply:
https://github.com/getsentry/relay/pull/1459